src/components/molecules/Monaco/Monaco.tsx

```diff
@@ -305,6 +306,13 @@ const Monaco = (props: {diffSelectedResource: () => void; applySelection: () =>
     } else {
       log.warn('[Monaco]: selected file was updated outside Monokle - unable to read file');
     }
+
+    if (filePath.endsWith('.js')) {
```

Comment:
why are we hard-coding support for `.js` files?
I'm not sure if it's even going to work without installing the monaco language for javascript, so we should probably keep supporting only yaml for now